### ISSUE TITLE
set weights  to 1 

### ISSUE THREAD

**Issue by assafdv** [jonasrothfuss/fishervector#7]
Hi, 
can you explain why you set the gmm weights to 1 (FisherVector.py, line 170). it seems that the likelihood_ratio param should hold the posterior probabilities p(C|X) (dented as gamma in Perronnin et al.). For my understanding this is what you get when you call perdict_proba. 

thanks, 
Assaf.

**Comment by qinchuanhui** [comment]
I agree with that, the averaged weights (with 1) may get much worse result.
